packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/SingleTextViewer/SingleTextViewerContainer.js

```diff
@@ -1,38 +1,52 @@
 import PropTypes from 'prop-types'
 import asyncStates from '@zooniverse/async-states'
 
+import { useTextData } from '@helpers'
```

Comment:
This should really go into `src/hooks`, with the other custom hooks. It's aliased to `@hooks`.

Reply:
Oops, yes indeed!